tests/unit/s2n_cipher_suite_match_test.c

```diff
@@ -672,6 +685,46 @@ int main(int argc, char **argv)
                 EXPECT_SUCCESS(s2n_connection_wipe(conn));
             }
         }
+        {
+            /* For TLS1.3 connections when a chosen PSK is present, a cipher suite with matching 
+             * hash algorithm must be selected */ 
+            {
+                EXPECT_SUCCESS(s2n_enable_tls13());
```

Comment:
Don't use s2n_enable_tls13() anymore; it's deprecated. Use "s2n_reset_tls13()" or nothing.